polyhost/server/mpc_listener.py

```
@@ -0,0 +1,281 @@
+            # Endpoint already gone, refusing, or busy — nothing to wake.
+            pass
+
+    # ------------------------------------------------------------------
+    # Accept loop + per-connection handler
+    # ------------------------------------------------------------------
+
+    def _accept_loop(self):
+        while self._running:
+            try:
+                conn = self._listener.accept()
+            except Exception:
```

Ruff (0.16.1)

[warning] Do not catch blind exception: `Exception`

(BLE001)